metricbeat/module/http/server/_meta/docs.asciidoc

```diff
@@ -2,12 +2,15 @@ This is the server metricset of the module http.
 
 Events sent to the http endpoint will be put by default under the `http.server` prefix. To change this use the `server.paths`
 config options. In the example below every request to `/foo` will be put under `http.foo`.
-
+Also consider using secure settings for the server using TLS/SSL as shown
 ["source","yaml",subs="attributes"]
 ------------------------------------------------------------------------------
 - module: http
   metricsets: ["server"]
   host: "localhost"
+  ssl.certificate_authorities: ["/etc/pki/root/ca.pem"]
+  ssl.certificate: "/etc/pki/client/cert.pem"
```

Comment:
I think it would be better to not use the word "client" in the path. That could be confusing. How about replacing that with `server`.

Reply:
great